### PR TITLE
CGP dev environment automation

### PR DESCRIPTION
**Pull request by finiteops**

### CODE REVIEW

File: .gcp/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  volumes:
+      - db-volume:/chain-db
+
+services:
+  # Needs access to EVMC_URL (canister http endpoint address)
+  # Doesn't expose any ports
+  # Writes data to db-volume
+  blockimporter:
+    <<: *default-erigon-service
+    # network_mode: host
```
Comment (GraDKh) [inline .gcp/docker-compose.yaml]: Please remove commented code

File: .github/workflows/dev-publish.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        uses: 'google-github-actions/upload-cloud-storage@v1'
+        with:
+          path: '.gcp'
+          parent: false
+          destination: ${{ env.GC_BUCKET }}/erigon
+
+      - name: Building base Rust/DFX image
+        uses: docker/build-push-action@v4
+        with:
+          # push-image: ${{ (github.event_name != 'pull_request' && github.ref == 'refs/heads/evmc_importer') ||  github.ref_type == 'tag' }}
```
Comment (GraDKh) [inline .github/workflows/dev-publish.yml]: Please remove commented code

File: Dockerfile [hunk 3]
```diff
@@ -104,4 +108,5 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vendor="Torquem" \
       org.label-schema.version=$VERSION
 
-ENTRYPOINT ["erigon"]
+# Entrypoint is messing up with compose
```
Comment (GraDKh) [inline Dockerfile]: I'd rather override entry point in `x-erigon-service` in docker-compose to have as few changes in this file as possible

**Review by GraDKh — APPROVED** [review]
I'm ok with the changes. However if there is a way to make changes local within `blockimporter` folder - it would be great, less problems with future merges

File: Dockerfile (hunk 3 above)
Comment (finiteops) [reply Dockerfile]: Will try to do that

**Comment by finiteops** [comment]
I messed up with a rebasing here.

**Pull request closed without merging**